flaml/automl.py

```diff
@@ -823,6 +824,8 @@ def custom_metric(X_test, y_test, estimator, labels,
             y_val: None | a numpy array or a pandas series of validation labels
             sample_weight_val: None | a numpy array of the sample weight of
                 validation data
+            verbose : int, default=0 | Controls the verbosity, higher means more

```

Comment:
default=1?

Reply:
remove the extra space before colon

Reply:
So before adding the verbose arguments, 'INFO' statements were being printed (as if verbose is >0). but at the same time, when calling _search(...) , 0 was being passed to verbose. Now we either pick verbose default value to be 1 (so that we get those 'INFO' prints) or if we set verbose=0, then due to some previous test that checks debug buffer, we would fail that test, and then we would have to change that test.

Reply:
Please my last comment in the bottom.